documentation/docs/01-routing.md

```diff
@@ -51,11 +51,12 @@ Endpoints are modules written in `.js` (or `.ts`) files that export functions co
 // Declaration types for Endpoints
 // * declarations that are not exported are for internal use
 
-export interface RequestEvent<Locals = Record<string, any>> {
+export interface RequestEvent<Locals = Record<string, any>, Platform = Record<string, unknown>> {
```

Comment:
Maybe let's use `any` instead of `unknown` for now for consistency and then make any changes across the whole codebase when addressing https://github.com/sveltejs/kit/issues/3513#issuecomment-1022238157 so that we come up with a consistent set of guidelines for using `any` vs `unknown`